examples/kepler-integration/src/app.js

```diff
@@ -107,8 +107,8 @@ const WindowSize = styled.div`
 `;
 
 // Custom localizations needed for export video modal
-messages.en['exportVideoModal.edit'] = 'Edit';
-messages.en['exportVideoModal.export'] = 'Export';
+messages.en['exportVideoModal.animation'] = 'Animation';
+messages.en['exportVideoModal.export'] = 'Settings';
```

Comment:
should you also change export key to be `'exportVideoModal.settings'`

Reply:
Good catch! I see that I forgot to make that consistent in several other places as well. Fixed w/ latest commit